client/i18n/locales/english/motivation.json

```diff
@@ -552,10 +552,6 @@
       "quote": "That's the thing about people who think they hate computers. What they really hate is lousy programmers.",
       "author": "Larry Niven"
     },
-    {
-      "quote": "It had long since come to my attention that people of accomplishment rarely sat back and let things happen to them. They went out and happened to things.",
-      "author": "Leonardo da Vinci"
-    },
```

Comment:
There has been an update on the issue.
The author of this quote was Elinor.
Please make the updates mentioned here
https://github.com/freeCodeCamp/freeCodeCamp/issues/53376#issuecomment-1912603161

Reply:
Hello, hope you're doing well --  I have addressed your comment.